checker/examples/GradleExamples/GradleJava7Example/build.gradle

```diff
@@ -0,0 +1,65 @@
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+
+assert (JavaVersion.current().isJava7() || JavaVersion.current().isJava8Compatible())
+
+
+// Indicates the version of Java to which the source and class files should conform
+// If set to JavaVersion.VERSION_1_7 then type annotations must be in comments.
+ext.targetJavaVersion = JavaVersion.current().isJava7() ? JavaVersion.VERSION_1_7 : JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+
+configurations {
+    if (targetJavaVersion.isJava7()) {
+        checkerFrameworkJavac {
+            description = 'a customization of the Open JDK javac compiler with additional support for type annotations'
+        }
+    }
+    checkerFrameworkAnnotatedJDK {
+       description = 'a copy of JDK classes with Checker Framework type qualifers inserted'
+    }
+    checkerFramework {
+       description = 'The Checker Framework: custom pluggable types for Java'
+    } 
+}
+
+dependencies {
+    ext.checkerFrameworkVersion = '1.9.11'
+    ext.jdkVersion = JavaVersion.current().isJava7() ? 'jdk7' : 'jdk8'
```

Comment:
No, the annotated JDK version must match the `JavaVersion.current()` even if the user hard codes `targetJavaVersion` to something besides the current version.